command/command.go

```diff
@@ -415,10 +415,15 @@ func runChecker(checker *checks.Checker, checkReportCh chan *checks.Report, repo
 			if checker.Config.Action != nil {
 				env := []string{fmt.Sprintf("MACKEREL_STATUS=%s", report.Status), fmt.Sprintf("MACKEREL_PREVIOUS_STATUS=%s", lastStatus)}
 				go func() {
-					_, stderr, _, _ := checker.Config.Action.RunWithEnv(env)
+					logger.Debugf("Checker %q action: %q env: %+v", checker.Name, checker.Config.Action.CommandString(), env)
+					stdout, stderr, exitCode, err := checker.Config.Action.RunWithEnv(env)
 
 					if stderr != "" {
-						logger.Warningf("Checker %q action output stderr: %s", checker.Name, stderr)
+						logger.Warningf("Checker %q action stderr: %s", checker.Name, stderr)
+					} else if err != nil {
+						logger.Warningf("Checker %q action stdout: %q exitCode: %d error: %s", checker.Name, stdout, exitCode, err)
```

Comment:
When `err != nil`, the error should already be reported:
https://github.com/mackerelio/mackerel-agent/blob/e3fd9f9d4181f3b72740f083b4454c37521a4358/cmdutil/cmdutil.go#L91-L92